docker/openjdk-latest/Dockerfile

```diff
@@ -1,5 +1,6 @@
 
 FROM openjdk:slim-buster
+ARG VERSION="latest"
```

Comment:
One More Time (we will miss you Daft Punk) should this be `dev`?

Reply:
I definitely will miss Daft Punk. I am trying to find the proper reasoning as to why `latest`.

Reply:
OK, `dev` sounds good. Changing.